_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1541,118 +1453,161 @@ link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#man
+    }
+
+    bob = {
+      groups               = ["_account.stage-full-access", "_account.prod-read-only", "ssh-grunt-sudo-users"]
+      pgp_key              = "keybase:bob_on_keybase"
+      create_login_profile = true
+      create_access_keys   = false
+    }
 
-  # Only set this to true if, when running 'terragrunt destroy,' you want to delete the contents of the S3 bucket that
-  # stores the CloudTrail logs. Note that you must set this to true and run 'terragrunt apply' FIRST, before running 'destroy'.
-  force_destroy = false
+    carol = {
+      groups               = ["_account.stage-full-access"]
+      pgp_key              = "keybase:carol_on_keybase"
+      create_login_profile = true
+      create_access_keys   = false
+    }
+  }
 }
 ----
 
-As before, configure the backend you want to use by including the settings from the root `terragrunt.hcl`:
+The example above configures the security account of an AWS Organization (the one where all your IAM users and IAM groups are defined), including setting up
+AWS Config, AWS CloudTrail, Amazon GuardDuty, IAM users, IAM groups, IAM password policy, and more. We create a handful of IAM groups including a `full-access`
+group (solely for a few trusted admins), plus a number of groups that give specific permissions in all your other child accounts (e.g., full-access in dev,
+read-only in prod, etc). Create an IAM user for yourself in the `full-access` group, plus IAM users for the rest of your team in the appropriate groups.
+
+Like the root account, the code will also generate a password for each user and encrypt it with that user’s PGP key from Keybase (see below for how to handle
+the passwords). You should follow this pattern to create an IAM user for yourself, as well as the small number of other trusted admins at your company who should
+have access to the root account.
+
+Configure your Terraform backend:
 
-.infrastructure-live/root/_global/cloudtrail/terragrunt.hcl
+.infrastructure-live/security/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
 include {
   path = find_in_parent_folders()
 }
 ----
 
-Since you already deleted the root user's access keys, this time, you should authenticate as your IAM user in the root
-account, making sure to set the MFA token on the CLI. See
-https://blog.gruntwork.io/a-comprehensive-guide-to-authenticating-to-aws-on-the-command-line-63656a686799[A Comprehensive Guide to Authenticating to AWS on the Command Line]
-for instructions on how to do that.
-
-Finally, deploy the `cloudtrail` module by running `terragrunt apply`:
+https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_add-key[Create a set of access keys for the root user in the security account]
+and https://blog.gruntwork.io/a-comprehensive-guide-to-authenticating-to-aws-on-the-command-line-63656a686799[use those access keys to authenticate on the CLI].
+Finally, deploy the `iam` module by running `terragrunt apply`:
```

Comment:
```suggestion
Finally, deploy the `account-baseliine` module by running `terragrunt apply`:
```

Reply:
good find, but changing to `account-baseline`.